tests/integration/tier1/destructive/kernel-excluded-in-conf/test_kernel_excluded_in_conf.py

```diff
@@ -0,0 +1,25 @@
+import pytest
+
+from conftest import TEST_VARS
+
+
+@pytest.mark.parametrize("yum_conf_exclude", [["kernel", "kernel-core", "redhat-release-server"]], indirect=True)
```

Comment:
```suggestion
@pytest.mark.parametrize("yum_conf_exclude", [["kernel*", "kernel-core*", "redhat-release-server"]], indirect=True)
```
As discussed on Slack, we should also cover kernel-modules and kernel-core-modules.
Going with asterisk would be helpful then.
Given that, we can get away with just `kernel*` which would cover all of `kernel kernel-core kernel-modules kernel-core-modules`

Reply:
Right, changing it to `kernel*`